org.eclipse.jdt.core.manipulation/common/org/eclipse/jdt/internal/ui/propertiesfileeditor/PropertiesFileEscapes.java

```diff
@@ -167,7 +167,7 @@ public static String unescape(String s) throws CoreException {
 					// Read the xxxx
 					int value= 0;
 					if (x > len - 4) {
-						String exceptionMessage= Messages.format(PropertiesFileEditorMessages.PropertiesFileHover_MalformedEncoding, outBuffer.toString() + s.substring(x - 2));
```

Comment:
Is it worth removing `PropertiesFileEditorMessages.PropertiesFileHover_MalformedEncoding` (and the corresponding property file entry) now that nothing refers to it ?

Reply:
Removed it